certified-connectors/Scancloud/apiDefinition.swagger.json

```diff
@@ -1,12 +1,12 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Secplugs Scan",
-    "description": "Secplugs Scan for scanning Emails content, files contents and Urls, Secplugs scan is a platform which creates hybrid security platform using various security provider",
+    "title": "Scancloud",
+    "description": "Scancloud for scanning Emails content, files contents and Urls, Scancloud is a platform which creates hybrid security platform using various security provider",
```

Comment:
@Ajaatsecplugs can you please describe concisely the main purpose and value offered by your connector in the description? You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).